how-to/create/mobile_automation_setup.rst

```diff
@@ -73,9 +73,7 @@ match between the build and the hook that starts the build.
 
     1. Update ``scriptworker/constants.py`` with entries for your product. Search for
        locations where "fenix" or "firefox-tv" were set up, and add your product accordingly
-    2. In a separate commit, bump the minor version and add a changelog
-       entry
-       (`example <https://github.com/mozilla-releng/scriptworker/commit/55626556eaf3aebdcf6aba408757bc39b76a941a>`__)
+    2. In a separate commit, bump the minor version
```

Comment:
This is talking about the scriptworker changelog, right? I think the file name changed since that commit, but we probably still want to encourage use of the scriptworker HISTORY.rst?